charts/webapi/values.yaml

```diff
@@ -10,6 +10,8 @@ global:
     tag: latest
 
 replicaCount: 1
+aspnetcore:
+  environment: Staging
```

Comment:
What is this chart yaml file for? What environment?

Reply:
This is the default aspnetcore.environment value, it is overridden during deployment.